TLM/.editorconfig

```diff
@@ -1,9 +1,77 @@
-﻿root = true
-
-[*.{cs,vb}]
-indent_style = space
-indent_size = 4
-trim_trailing_whitespace = true
+root=true
 
 [*.cs]
-# csharp_new_line_before_open_brace = methods
+max_line_length=100
+tab_width=8
+
+[*]
+charset=utf-8
+end_of_line=crlf
+trim_trailing_whitespace=true
```

Comment:
The `trim_trailing_whitespace` will adversely affect `.md` files that use two trailing spaces to force a newline. I think we should move the setting to the `.cs` section of the file.

Reply:
Moved up to .cs